srcpkgs/protontricks/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'protontricks'
+pkgname=protontricks
+version=1.2.4
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-setuptools"
+depends="python-setuptools vdf winetricks"
```

Comment:
if it requires setuptools as an entry point, it should also be the version for python3 here.

Reply:
Looks like it doesn't actually require setuptools as a hostmakedepend or as a depend, so I've removed both.